docs/30_commands.md

```diff
@@ -188,3 +196,93 @@ Available flags for the command:
 - `--context`, to specify a different context from the currently selected one
 - `--company-id`, to set the ID of the desired company
 - `--project-id`, to set the ID of the desired project
+
+## marketplace
+
+View and manage Marketplace items
+
+All the subcommands inherit the following flags:
+
+```
+      --auth-name string               the name of the miactl auth to use
+      --certificate-authority string   path to a cert file for the certificate authority for the selected endpoint
+      --company-id string              the ID of the company
+  -c, --config string                  path to the config file default to $HOME/miactl/config
+      --context string                 the name of the miactl context to use
+      --endpoint string                the address and port of the Mia-Platform Console server
+      --insecure-skip-tls-verify       if true, the server's certificate will not be checked for validity. This will make your HTTPS connections insecure
+      --verbose                        increase the verbosity of the cli output
+```
+
+### list
+
+List Marketplace items
+
+#### Synopsis
+
+List the Marketplace items that the current user can access.
+
+```
+miactl marketplace list [flags]
+```
+
+### get
+
+Get a Marketplace item
+
+#### Synopsis
+
+Get a single Marketplace item by its ID
+
+```
+miactl marketplace get resource-id [flags]
+```
+
+### delete
+
+Delete a Marketplace item
+
+#### Synopsis
+
+Delete a single Marketplace item by its ID
+
+```
+miactl marketplace delete resource-id [flags]
+```
+
+### apply
+
+Create or update Marketplace items
+
+#### Synopsis
+
+Create or update one or more Marketplace items.
+
+You can either specify: - one or more files, with the flag -f - one or more directories, with the flag -d
+Supported formats are JSON (.json files) and YAML (.yaml or .yml files).
+
+```
+miactl marketplace apply { { -f file-path }... | { -d directory-path }... } [flags]
+```
+
+#### Examples
+
+```
+
+# Apply the configuration of the file myFantasticGoTemplate.json located in the current directory to the Marketplace
+miactl marketplace apply -f myFantasticGoTemplate.json
+
+# Apply the configurations in myFantasticGoTemplate.json and myFantasticNodeTemplate.yml to the Marketplace, with relative paths
+miactl marketplace apply -f ./path/to/myFantasticGoTemplate.json -f ./path/to/myFantasticNodeTemplate.yml
+
+# Apply all the valid configuration files in the directory myFantasticGoTemplates to the Marketplace
+miactl marketplace apply -d myFantasticGoTemplates
+```
+
+#### Options
+
+```
+  -d, --directory string   a path to a directory containing Marketplace resource files
+  -f, --file stringArray   a path to a JSON or YAML file containing a Marketplace resource
```

Comment:
@malta895, are these options still correct, or is an update needed?

Reply:
Yes an update is needed. I'll integrate it with the image upload functionality, as soon as it is finished